### ISSUE TITLE
Failed to instantiate provider "aws" to obtain schema: timeout while waiting for plugin to start

### ISSUE THREAD

**Issue by brianpham** [gruntwork-io/terragrunt#1164]
Error: ```Failed to instantiate provider "aws" to obtain schema: timeout while waiting for plugin to start```

When I run, ```terragrunt validate-all``` I am seeing the following error above. It seems to work for 90% of the regions, but will error out at the very end. Any ideas on what might be going on? 

terragrunt 0.23.13
terraform 0.12.24

**Comment by yorinasub17** [comment]
My guess is that this is related to https://github.com/gruntwork-io/terragrunt/issues/879. How many components are in your terragrunt stack that you are running `validate-all` on, and how many of those are parallel?

**Comment by brianpham** [comment]
@yorinasub17 About 30 components and all are running in parallel. I am going to try to split it up to see if it helps.

**Comment by brianpham** [comment]
@yorinasub17 Yeah, that did the trick. I will close this issue, but it is definitely related #879
